### DISCUSSION TITLE
"Free_Fonts.h" not found, example code works fine

### DISCUSSION THREAD

**Discussion by Rhythm8503** [Bodmer/TFT_eSPI#2918] [Q&A - Fonts] [answered]
This is mainly a library issue, for some reason the "Free_Font.h" is not being found, I ran an example sketch to test but it compiled fine, when I run my code with the Free_Fonts it doesn't compile since the Arduino IDE cannot find the file/directory. Any solution to this?
This is on the latest update of the TFT_eSPI, using the uPesy ESP32 WROOM Kit, with an ili9486 on parallel.

**Comment by Bodmer** [comment, marked as answer]
The Free_Font.h file has to be put in the sketch folder. It will appear as an extra code tab:
![image](https://github.com/Bodmer/TFT_eSPI/assets/15824805/7fd3f02f-eab8-4fa6-8fa6-3a316a29b558)
